chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/TestProcedure.java

```diff
@@ -2,14 +2,36 @@
 
 import java.io.Serializable;
 
-public class TestProcedure extends KeyValueModel implements Serializable {
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlType;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+
+@XmlType(namespace = "http://chpl.healthit.gov/listings")
+@XmlAccessorType(XmlAccessType.FIELD)
+@JsonIgnoreProperties(ignoreUnknown = true)
+@AllArgsConstructor
+@NoArgsConstructor
+public class TestProcedure implements Serializable {
     private static final long serialVersionUID = -3763885258251744916L;
 
-    public TestProcedure() {
-        super();
-    }
+    private Long id;
+    private String name;
 
-    public TestProcedure(Long id, String name) {
-        super(id, name);
+    public Long getId() {
```

Comment:
Could these be handled with the `@Data` Lombok annotation? I have a vague memory that our XML things can't use those annotations for some reason, but wanted to double check

Reply:
I also recall that we can't use @Data in classes that are involved in the XSD/XML stuff. That's why I didn't.